env/env_encryption_ctr.h

```diff
@@ -10,31 +10,6 @@
 #include "rocksdb/env_encryption.h"
 
 namespace ROCKSDB_NAMESPACE {
-
-// Implements a BlockCipher using ROT13.
-//
-// Note: This is a sample implementation of BlockCipher,
-// it is NOT considered safe and should NOT be used in production.
-class ROT13BlockCipher : public BlockCipher {
```

Comment:
why we move it out from the header file?

Reply:
There is no need for it in the header file.  The class does not need to be public